projects/demo/src/pages/kit/date-time/examples/4-strict-mode/component.ts

```diff
@@ -0,0 +1,39 @@
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+import {FormsModule} from '@angular/forms';
+import {MaskitoDirective} from '@maskito/angular';
+import {TuiTextfieldControllerModule} from '@taiga-ui/core';
+import {TuiInputModule} from '@taiga-ui/kit';
+
+import mask from './mask';
+
+@Component({
+    standalone: true,
+    selector: 'date-time-mask-doc-example-4',
+    imports: [
+        TuiInputModule,
+        TuiTextfieldControllerModule,
+        FormsModule,
+        MaskitoDirective,
+    ],
+    template: `
+        <tui-input
+            tuiTextfieldCustomContent="tuiIconCalendarLarge"
+            [style.max-width.rem]="30"
+            [tuiTextfieldFiller]="filler"
+            [(ngModel)]="value"
+        >
+            Min-max
```

Comment:
Why the such label ?

Reply:
I was copy-pasting things, and forgot rename it🫣